obsolete/LDR_photocell_300mil_v4.fzp

```diff
@@ -13,7 +13,7 @@
     </tags>
     <properties>
         <property name="resistance@ dark">300 kOhms@ 10 seconds</property>
-        <property name="family">Photo-Resistor</property>
+        <property name="family">//obsolete//Photo-Resistor</property>
```

Comment:
Which change required the obsoletion?  I think here the only change is that the SPICE field was added. Now, the question is, should we obsolete the previous part because of that? I think Fritzing will just pick up the change the next time a project using that part is loaded, no obsoletion needed.

Reply:
Similar to the other case. The id has changed and now it ends in "PhotocellModuleID". This makes possible to change the amount of light received and other properties of the photocell.